subiquity/common/apidef.py

```diff
@@ -121,10 +121,12 @@ def POST(tty: str) -> None:
                 """Confirm that the installation should proceed."""
 
         class restart:
+            @allowed_before_start
```

Comment:
Yeah to enable the new dialog flow. Crashing before finishing controller setup means we'll be blocked on most endpoints. 

The only instance of `meta.restart.POST` in the client is the one this change is fixing, and yes no other dependencies.

There's a couple points where `meta.ssh_info.GET` is called, which does the require two things:
- The cloud init stage to finish. 
  -  Which it looks like the clients, the server client at least, wait to fully start until after this stage. So password info from cloud init should be set
- The network model to be configured. 
  - Looking into this one actually it looks like the network model won't be fully configured by time it's called in the error state. Essentially making it useless I think. What if we add some logic to detect the early fail case and load the network controller, otherwise block until it's configured?